package.json

```diff
@@ -33,7 +33,7 @@
   "homepage": "https://github.com/TokenySolutions/T-REX#README",
   "devDependencies": {
     "@commitlint/cli": "^8.3.5",
-    "@onchain-id/solidity": "0.3.1",
+    "@onchain-id/solidity": "0.4.0",
```

Comment:
there is a problem with this dependency apparently ...

Reply:
I feel so dumb >>'
The published package had the wrong version of the repo, I have released a new version 0.4.1 with the proper contracts, sorry!